galleries/examples/shapes_and_collections/ellipse_arrow.py

```diff
@@ -0,0 +1,113 @@
+"""
+===================================
+Ellipse with orientation arrow Demo
+===================================
+
+This demo shows how to draw an ellipse with
+an orientation arrow (clockwise or counterclockwise).
+Compare this to the :doc:`Ellipse collection example
+</gallery/shapes_and_collections/ellipse_collection>`.
+"""
+
+from typing import Tuple
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from matplotlib.markers import MarkerStyle
+from matplotlib.patches import Ellipse
+from matplotlib.transforms import Affine2D
+
+# %%
+#
+# A method to calculate the end points of the ellipse major, minor axis
+# """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+#
+# Calculates the minor axis and major axis end points of the ellipse.
+# It needs the ellipse parameter like width, height, angle.
+# The output are 2 lists of 2 xy coordinates
+# (minor((x0, y0), (x1, y1)), major((x0, y0), (x1, y1))).
+
+
+def getMinorMajor(ellipse: Ellipse) -> Tuple[list, list]:
```

Comment:
In this particular example, you are  only using `x0_minor[0]`, so you can simplify the code. Note that the technical term for this point is *co-vertex*, see https://en.wikipedia.org/wiki/Ellipse.

That said, I can see the case for adding `Ellipse.get_vertices()` and `Ellipse.get_co_vertices()` methods - but this would need proper documentation and testing, so is a bit more effort.

Reply:
Do you also see a marker property of Ellipse Patches?

Reply:
No markers would be too specialized. In general, ellipses do not have anything to do with markers. In contrast, vertex/co-vertex are core properties of an ellipse, which we might provide as convenience.